docs/tutorials/index.rst

```diff
@@ -0,0 +1,49 @@
+GeoMesa |release| Tutorials
+===========================
+
```

Comment:
The tutorials expect geomesa-tutorials to be available.

I don't think the manual mentions ``geomesa-ext`` at all. Since the GeoServer plugins are still in the main GeoMesa distribution, I don't think I understand which bits of functionality were pulled out of GM and put into GM-ext.